javascript/ql/test/query-tests/Security/CWE-079/unsafe-jquery-plugin.js

```diff
@@ -152,9 +152,9 @@
 
 	$.fn.my_plugin = function my_plugin(options) {
 		let target = options.target;
-		target === DEFAULTS.target? $(target): $(document).find(target); // NOT OK
-		options.target === DEFAULTS.target? $(options.target): $(document).find(options.target); // NOT OK
-		options.targets.a === DEFAULTS.target? $(options.target.a): $(document).find(options.target.a); // OK - but still flagged
+		target === DEFAULTS.target? $(target): $(document).find(target); // NOT OK - but not flagged [INCONSISTENCY]
+		options.target === DEFAULTS.target? $(options.target): $(document).find(options.target); // NOT OK - but not flagged [INCONSISTENCY]
+		options.targets.a === DEFAULTS.target? $(options.target.a): $(document).find(options.target.a); // OK - but still flagged [INCONSISTENCY]
```

Comment:
@esbena: I found a bunch of inconsistencies in `unsafe-jquery-plugin.js`.

Reply:
I think this should be:


```suggestion
		target === DEFAULTS.target? $(target): $(document).find(target); // OK
		options.target === DEFAULTS.target? $(options.target): $(document).find(options.target); // OK
		options.targets.a === DEFAULTS.target? $(options.target.a): $(document).find(options.target.a); // OK - but still flagged [INCONSISTENCY]
```

I think our relatively recent membership tests sanitizers changed our precision.